CHANGELOG.md

```diff
@@ -2,8 +2,43 @@
 
 ## Unreleased
 
+- [THIS PR](link to PR) BREAKING CHANGE: Mostly fixes and changes to types, but some method's now accept slightly different parameters:
+
+  - `createPaymentMethod`'s `billingDetails` and `shippingDetails` parameters no longer accept the `addressPostalCode`, `addressCity`, `addressCountry`, `addressLine1`, `addressLine2`, or `addressState` keys. Instead, they accept an `address` object containing: `city`, `country`, `line1`, `line2`, `postalCode`, `state`.
+  - `confirmPayment`'s `billingDetails` and `shippingDetails` parameters no longer accept the `addressPostalCode`, `addressCity`, `addressCountry`, `addressLine1`, `addressLine2`, or `addressState` keys. Instead, they accept an `address` object containing: `city`, `country`, `line1`, `line2`, `postalCode`, `state`.
+  - `Card.Token` is now `Token.Result`
+  - `Card.Brand` is now `Token.CardBrand`
+  - `Card.TokenType` is now `Token.Type`
+  - `Card.BankAccount` is now `Token.BankAccount`
+  - `Card.Params` is now `Token.Params`
+  - `CardFormView.Names` is now `CardFormView.FieldName`
+  - `CardFieldInput.Names` is now `CardFieldInput.FieldName`
+  - `ApplePayButtonComponent.Styles` is now `ApplePayButtonComponent.Style`
+  - `ApplePayButtonComponent.Types` is now `ApplePayButtonComponent.Type`
+  - `PaymentMethod` is now `PaymentMethod.Result`
+  - `PaymentIntent` is now `PaymentIntent.Result`
+  - `SetupIntent` is now `SetupIntent.Result`
+  - `Card.Token` is now `Token`
```

Comment:
On line 9 you say `Card.Token` is now `Token.Result`, but on this line you say `Card.Token` is now `Token` which is correct?

Also, `Token` is a super overloaded term in Stripe. Is this actually a type able to encapsulate all past & future `Token` types? or is it only acceptable for `CardTokens`?

Reply:
Woops, my bad. "`Card.Token` is now `Token`" should read that exports that were under the `Card` namespace are now under the `Token` module

Reply:
Currently, it represents either a cardtoken or a bankaccount token - https://github.com/stripe/stripe-react-native/pull/837/files#diff-3e9aecf5a36afc03b7f6331ddd4627a9b468a437423687da1c47dcec88ab3eb2R1-R71